src/main/webapp/js/instructorStudentList.es6

```diff
@@ -323,6 +324,12 @@ const seeMoreRequest = function (e) {
         // Do not make ajax call if students shown already above limit
         showStudentLimitError(courseCheck, displayIcon);
     }
+
+    if ($(panelCollapse).attr('class').indexOf('checked') === -1) {
+        $(toggleCheveron).addClass('glyphicon-chevron-down').removeClass(' glyphicon-chevron-up');
+    } else {
+        $(toggleCheveron).addClass('glyphicon-chevron-up').removeClass(' glyphicon-chevron-down');
```

Comment:
Remove the space in `glyphicon-chevron-down`. Similarly for `glyphicon-chevron-up` up there.

Also, see if there are chevron-related JS functions defined somewhere that you can reuse.